TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/teleop/DriveTeleop.java

```diff
@@ -31,16 +32,24 @@ public void loop(){
             intakeSystem.stop();
         }
 
+        if (gamepad1.x && !leftLatchHit) {
```

Comment:
The FTC code actually has a debounce-esque class `Deadline` for handling this situation. See `SampleRevBlinkinLedDriver.java` for example usage.